core/src/main/java/org/wso2/testgrid/core/command/RunTestPlanCommand.java

```diff
@@ -74,6 +74,11 @@
             required = true)
     private String workspace = "";
 
+    @Option(name = "--url",
+            usage = "Jenkins URL",
+            aliases = {"-u"})
```

Comment:
Make this optional?

Reply:
the default value for the required parameter is false, so this is optional. are you suggesting to make this mandatory?

Reply:
Just wanted to make sure this is optional. (y)